ogr/mock_core.py

```diff
@@ -117,3 +122,58 @@ def pr_merge(cls, original_object: Any, pr_id: int) -> "PullRequest":
     @classmethod
     def fork_create(cls, original_object: Any) -> "GitProject":
         return original_object
+
+
+class PersistentObjectStorage:
+    storage_file: str = ""
+    storage_object: dict = {}
+    write_mode: bool = False
+
+    def __init__(self, storage_file: str, write_mode: Optional[bool] = None) -> None:
+        self.storage_file = storage_file
```

Comment:
what's the semantic behind this attribute?

Reply:
to have stored file what is used for reading and writing. `dump` and `load` methods uses it, to load or store data to file what is defined in contsructor. currently it is not used much, but my idea there were, that you can also explicitly call load and dump methods, whenever you want and not define eg. file parameter in  each dump method. and also it allows me, to use this file name param within __del__ to call dump when object dissappear.
Or you meant something else what I should describe